transport-native-epoll/src/main/java/io/netty/channel/epoll/NativeFileDescriptor.java

```diff
@@ -19,11 +19,15 @@
 
 import java.io.IOException;
 
-final class EpollFileDescriptor implements FileDescriptor {
+/**
+ * Native {@link FileDescriptor} implementation which allows to wrap an {@code int} and provide a
+ * {@link FileDescriptor} for it.
+ */
+public final class NativeFileDescriptor implements FileDescriptor {
```

Comment:
Just a note - when you introduce `netty-transport-native-common`, we might want to merge `NativeFileDescriptor` into `FileDescriptor`.